applications/matter_bridge/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# TODO: Workaround to be removed once DFU and external flash will be supported on nRF54H20.
+CONFIG_CHIP_OTA_REQUESTOR=n
+CONFIG_CHIP_QSPI_NOR=n
+
+CONFIG_MPU_STACK_GUARD=n
+
+# TODO: Enable PSA crypto once it is available
+CONFIG_CHIP_CRYPTO_PSA=n
+CONFIG_MBEDTLS_CIPHER_PADDING_PKCS7=y
+CONFIG_MBEDTLS_CIPHER_MODE_CBC=y
+
+# TODO: Enable factory data once it is available
+CONFIG_CHIP_FACTORY_DATA=n
+
+# TODO: Workaround to be removed once Zephyr's CONFIG_FPROTECT is supported on nRF54H20.
+CONFIG_CHIP_FACTORY_DATA_WRITE_PROTECT=n
+
+# Set SUIT template paths
+CONFIG_SUIT_ENVELOPE_TEMPLATE="suit_templates/app_envelope.yaml.jinja2"
```

Comment:
Looking at the PR from Arek with common suit templates, I think this is not required, but please double-check.